cypress/integration/identifier/PersistentLinks.feature

```diff
@@ -0,0 +1,41 @@
+Feature: Persistent Links
+  As a user
+  I want all links to carry permanent information that identifies the linked resource
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-03-26T23:04:11Z" title="Wednesday, March 27th 2019, 12:04:11 am +01:00">Mar 27, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,41 @@
+Feature: Persistent Links
+  As a user
+  I want to click on a link with an 'href' having an unmodifiable id of a user
```


I doubt this is how a user of the site would think about it, but maybe I am just being pedantic.... 
I would expect them to think more along the lines of `I want to avoid clicking on links that lead nowhere or an error page`